### ISSUE TITLE
tag v2.0.11: build script in archive fails looking for git revision

### ISSUE THREAD

**Issue by steveeJ** [coreos/etcd#2867]
When building from the released archive for v2.0.11 the build fails with the following error

```
etcd-2.0.11 # ./build 
fatal: Not a git repository (or any of the parent directories): .git
```

**Comment by xiang90** [comment]
yea... It is trying to get the git tag. I will look into it.

**Comment by xiang90** [comment]
@steveeJ Thank you for reporting this. We have found the root cause and changed our release process. This will be fixed in 2.0.12+. Thanks!
